apache/superset
# fix(export_dashboards): update role model.  phase 1

### SUMMARY

#### Enable Export Dashboards with Associated Roles
#### Bug: 
export/import dashboards doesn't support roles. see here [issue](https://github.com/apache/superset/issues/17564)

#### Solving:
In order to add roles [which are associated  to dashboards]  to the export_dashboards functionality,    
(Via cli or http request),  
we need that the role model will inherit from the ImportExportMixin class.

changes:  
1. Role model - inherit from ImportExportMixin.
2. User Model - update the role relation

  
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS

All test should pass

In order to enable dashboard RBAC we need to add enable 2 feature flag:

FEATURE_FLAGS =

{ "VERSIONED_EXPORT": True, "DASHBOARD_RBAC": True, } 

### ADDITIONAL INFORMATION


- [ X] Has associated issue:
- [ X] Required feature flags:
